core/src/main/resources/lib/layout/html.jelly

```diff
@@ -147,6 +147,7 @@ ${h.initPageVariables(context)}
       <link rel="search" type="application/opensearchdescription+xml" href="${rootURL}/opensearch.xml" title="Jenkins" />
     </l:hasPermission>
     <meta name="ROBOTS" content="INDEX,NOFOLLOW" />
+    <meta name="viewport" content="width=device-width, initial-scale=1" />
```

Comment:
Not sure why git showed it as part of the diff, but I've merged the upstream master branch to my working branch and it has fixed the view of the diff now